packages/cli/src/deploy-create2/DeployCreate2Command.tsx

```diff
@@ -0,0 +1,390 @@
+import {Box, Text} from 'ink';
```

Comment:
main flow

1. runs checks (preverification and simulation)
2. once all checks are run (the container component uses useChecksForChains to stay reactive to each of the individual chain's check states with caching), it renders an option to deploy using a private key or by connecting a wallet. if privateKey is already passed, it skips this interactivity and just immediately deploys
3. Each of the individual chains also shows the state of the checks